src/zone.ts

```diff
@@ -56,25 +101,44 @@ export function updateAddresses(coin: string, text: any | string, addresses: str
         })
     }
     zoneFileJson.txt = txts;
-    return zoneFileJson;
+}
+
+export function parseZoneFile(zoneFile: string) {
```

Comment:
Should we create a type for the return type here?

Reply:
sounds good - copied over the types from the stacks zone-file lib